app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java

```diff
@@ -1223,10 +1220,7 @@ public void updateMarker(boolean isBookmarked, Place place, @Nullable fr.free.nr
      * but it is transparent more than grey(as the name of the icon might suggest)
      * since grey icon may lead the users to believe that it is disabled or prohibited contribution
      */
-    private void hideAllMArkers() {
-        if(currentLocationMarker==null){
```

Comment:
Why this nullceck is removed?

Reply:
It prevents initial filtering when Needs Photos chip set.
Check [this](https://github.com/commons-app/apps-android-commons/issues/3704)